src/plugins/network.ts

```diff
@@ -117,15 +118,25 @@ export class NetworkManager {
       step: this._currentStep,
       timestamp: getTimestamp(),
       url,
-      request,
       type,
       method,
-      requestSentTime: timestamp,
+      status: -1,
+      request: {
+        url,
+        method,
+        headers,
+        bytes: postData?.length || 0,
+      },
+      response: {
+        statusCode: -1,
```

Comment:
Why do we set this to `-1` vs `null`

Reply:
explained below. https://github.com/elastic/synthetics/pull/467#discussion_r823293893